channeldb/graph_test.go

```diff
@@ -2680,6 +2680,103 @@ func TestNodeIsPublic(t *testing.T) {
 	)
 }
 
+// TestDisableChannelIDs ensures that the disabled channels within the
+// disabledEdgePolicyBucket are managed properly and the list returned from
+// DisabledChannelIDs is correct.
+func TestDisableChannelIDs(t *testing.T) {
+	t.Parallel()
+
+	db, cleanUp, err := makeTestDB()
+	defer cleanUp()
```

Comment:
`cleanUp()` should only be deferred if `err != nil`

Reply:
Fixed.